releaseNotes/Microsoft.Featuremanagement.AspNetCore.md

```diff
@@ -2,6 +2,16 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0-preview4 - Jul 19, 2024
+
+* Updated `Microsoft.FeatureManagement` reference to `4.0.0-preview4`. See the [release notes](./Microsoft.Featuremanagement.md) for more information on the changes.
+
+## 3.5.0 - Jul 19, 2024
+
+### Enhancements
+
+* Developers will now have a new extension method `WithTargeting()` which registers a default `ITargetingContextAccessor`. This default accessor will extract the targeting info from `HttpContext.User`. `UserId` will be taken from the Identity.Name field. `Groups` will be extracted from claims of type `Role`. [#466](https://github.com/microsoft/FeatureManagement-Dotnet/pull/466)
```

Comment:
Not sure if there is a concept of an overloaded extension method, but we should call it out to differentiate from the one in the base library. Please correct the terminology as appropriate. Customers should refer to documentation for implementation details.

Introduced an overloaded extension method `WithTargeting()` for ASP.NET Core applications. It registers a default `ITargetingContextAccessor` that constructs user context based on the authenticated user of a request.

Reply:
Two `WithTargeting` methods are in different static classes, technically, they are not overloaded methods.

But I think for customers, it is ok to treat them as overloaded extension methods.